### PR TITLE
fix(HACBS-2471): create tag based on full git sha

### PR DESCRIPTION
**Pull request by scoheb**
- Create tag based on full git sha

### CODE REVIEW

**Review by johnbieren — COMMENTED** [review]
Shouldn't this be 0.10, not 1.0?

**Comment by scoheb** [comment]
> Shouldn't this be 0.10, not 1.0?

fixed

File: catalog/task/push-snapshot/0.10/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+| snapshotPath    | Path to the JSON string of the mapped Snapshot spec in the data workspace                       | Yes      | mapped_snapshot.json |
+| tag             | Default tag to use if mapping entry does not contain a tag                                      | Yes      | latest               |
+| retries         | Retry copy N times                                                                              | Yes      | 0                    |
+| addGitShaTag    | Also push a tag with the git sha for each image in the Snapshot                                 | Yes      | true                 |
+| addSourceShaTag | Also push a tag with the source sha for each image in the Snapshot                              | Yes      | true                 |
+| addTimestampTag | Also push a tag with the current timestamp for each image in the Snapshot                       | Yes      | false                |
+
+## Changes since 0.9
+* `addGitShaTag` now creates 2 tags:
+  * tag composed of a shortened git sha based on 1st seven characters of sha (existing functionality)
```
Comment (johnbieren) [inline catalog/task/push-snapshot/0.10/README.md:18]: can you write out first instead of 1st?

**Review by johnbieren — DISMISSED** [review]

File: catalog/task/push-snapshot/0.10/README.md (hunk 1 above)
Comment (scoheb) [reply catalog/task/push-snapshot/0.10/README.md]: fixed

**Review by johnbieren — APPROVED** [review]

**Pull request merged**